public/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <style>
+      @import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
       @import url('https://fonts.googleapis.com/css2?family=Damion&display=swap');
```

Comment:
This is using CSS Syntax, but isn't there an HTML import syntax also? like a 1 out of 10 important, but if it's easy let's do that